crates/holochain/src/core/ribosome/host_fn/zome_info.rs

```diff
@@ -72,6 +72,7 @@ pub mod test {
                 FunctionName::new("__allocate"),
                 FunctionName::new("__data_end"),
                 FunctionName::new("__deallocate"),
+                FunctionName::new("__getrandom_custom"),
```

Comment:
Why is this function exported?

Reply:
Something to do with how the getrandom crate internally hooked up the glue when doing their wasm-unknown-unknown implementation. We can look at their code if you want, but I'm not sure there's much we can do about it.